modules/distr-tracing-tempo-install-tempostack-web-console.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-install-tempostack-web-console_{context}"]
+= Installing a TempoStack instance by using the web console
+
+You can install a TempoStack instance from the *Administrator* view of the web console.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a cluster administrator with the `cluster-admin` role.
+
+* For {product-dedicated}, you must be logged in using an account with the `dedicated-admin` role.
+
+* You have completed setting up the required object storage by a supported provider: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], link:https://cloud.google.com/storage/[Google Cloud Storage]. For more information, see "Object storage setup".
++
+[WARNING]
+====
+Object storage is required and not included with the {TempoShortName}. You must choose and set up object storage by a supported provider before installing the {TempoShortName}.
+====
+
+.Procedure
+
+. Create a project of your choice for the TempoStack instance that you will create in a subsequent step: go to *Home* -> *Projects* -> *Create Project*.
```

Comment:
```suggestion
. Create a project of your choice for the TempoStack instance that you will create in a subsequent step by clicking *Home* -> *Projects* -> *Create Project*.
```

Reply:
TY. I'll apply the same solution here as in the other comment.

Reply:
This is how I rephrased it:

`. Go to *Home* -> *Projects* -> *Create Project* to create a project of your choice for the TempoStack instance that you will create in a subsequent step.`